src/signals/incident-management/containers/IncidentStatusContainer/components/Add/index.js

```diff
@@ -53,6 +55,11 @@ class Add extends React.Component { // eslint-disable-line react/prefer-stateles
 
                   <button className="action primary" type="submit" disabled={invalid}>
```

Comment:
Do we also want to disable the button when (loading === true)?

Reply:
yes i will add that